webapp/src/main/resources/templates/frontend/index.html

```diff
@@ -0,0 +1,209 @@
+<!DOCTYPE html>
+<html layout:decorate="~{frontend/fragments/layout}"
+      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      lang="en" xmlns:th="http://www.w3.org/1999/xhtml">
+<head>
+    <title>Home</title>
+    <script type="module" th:src="@{/ts/dist/home.bundle.js}" src="#"></script>
+</head>
+<body>
+    <section layout:fragment="jumbotron">
+        <!-- TEMPLATES -->
+        <template id="artist-template">
+            <div class="swiper-item swiper-slide">
+                <img src="#" alt="" id="artist-image" class="img-round">
+                <div class="mt-2">
+                    <span class="swiper-title" id="artist-name"></span>
+                </div>
+                <div>
+                    <span class="text-muted font-size-s" id="follower"></span>
+                </div>
+            </div>
+        </template>
+        <template id="release-template">
+            <div class="swiper-item swiper-slide">
+                <img src="#" alt="" id="release-cover" class="img-round">
+                <div class="mt-2">
+                    <span class="swiper-title" id="release-artist"></span>
+                </div>
+                <div>
+                    <span class="swiper-subtitle font-size-s" id="release-title"></span>
+                </div>
+                <div class="text-muted font-size-s" id="release-date"></div>
+            </div>
+        </template>
+        <!-- TEMPLATES END -->
+
+        <div class="jumbotron">
+            <div class="container jumbotron-text pt-4 pb-4">
+                <h1>Built for <br />Metal Heads.</h1>
+                <p>
+                    Metal Detector is a metal release calendar that is 100% tailored to you.
+                    Follow your favorite artists and get notified when they announce or
+                    release a new album.
+                </p>
+                <a class="btn btn-white mt-3" href="#" th:href="@{/releases}">Explore Upcoming Releases</a>
+            </div>
+        </div>
+    </section>
+    <section layout:fragment="content">
+        <div id="homepage-wrapper">
+            <div class="mt-5 section">
+                <h2>About</h2>
+                <div class="headline-separator"></div>
+                <div class="features">
+                    <div class="feature-item">
+                        <i class="material-icons md-42 md-light">search</i>
+                        <p class="feature-text mt-2">
+                            Search and follow your favorite bands to
+                            stay up to date when a new album is coming out.
```

Comment:
```suggestion
                            stay up to date when a new album is issued.
```
"Coming out" sounds like very German translation. Also this can be misunderstood quite funnily 😉